homeassistant/components/media_player/plex.py

```diff
@@ -132,11 +149,16 @@ def setup_plexserver(
 
     plex_clients = {}
     plex_sessions = {}
+
+    for device in hass.data[PLEX_DATA].devices:
```

Comment:
Don't do this. Your data object should be your single source of truth. Don't keep a 2nd registry of entities